trie/node.go

```diff
@@ -271,14 +272,22 @@ func shouldStopIfContextDoneBlockingIfBusy(ctx context.Context, idleProvider Idl
 	}
 }
 
-func treatCommitSnapshotError(err error, hash []byte, missingNodesChan chan []byte) {
+func treatCommitSnapshotError(err error, hash []byte, missingNodesChan chan []byte) (bool, error) {
+	if err == nil {
+		return false, nil
+	}
+
+	if !strings.Contains(err.Error(), core.GetNodeFromDBErrorString) {
+		return false, err
```

Comment:
is this ok to return here false? I mean, the error is not of type `GetNodeFromDBErrorString` but is still an error & the node could have not been received.